Shared/Services/LoopringGraphQLService.cs

```diff
@@ -1499,6 +1499,60 @@ public async Task<int> GetAccountTotalNfts(string accountId, int chunkSize = 100
             }
         }
 
+        public async Task<List<NftHolder>> GetNftHolders(string nftId, int skip = 0, int first = 25, CancellationToken cancellationToken = default)
+        {
+            var nftHolders = @"
+            query nftHolders(
```

Comment:
In my testquery I ordered by Id, but I have forgotten to put that in there :) I would also think that it would be nice to order by balance, so I will add that as default parameter. And if specified by something else that would also work.

Reply:
Fixed :)